client/executor/runtime-test/src/lib.rs

```diff
@@ -56,7 +61,7 @@ sp_core::wasm_export_functions! {
 
 	fn test_empty_return() {}
 
-	fn test_exhaust_heap() -> Vec<u8> { Vec::with_capacity(16777216) }
+	fn test_exhaust_heap() -> Vec<u8> { Vec::with_capacity(18 * 64 * 1024) }
```

Comment:
The new value is `1179648`. I wonder why would we need such change?

Reply:
Ahh, I can probably revert this. Do you remember this question I had about initial pages?

I first had an implementation that used memory pages == heap pages, but now it is memory pages == intial + heap pages.

Tdlr, I revert this.